src/login/logind-inhibit.c

```diff
@@ -503,6 +505,8 @@ InhibitWhat inhibit_what_from_string(const char *s) {
                         what |= INHIBIT_HANDLE_HIBERNATE_KEY;
                 else if (l == 17 && strneq(word, "handle-lid-switch", l))
                         what |= INHIBIT_HANDLE_LID_SWITCH;
+                else if (l == 18 && strneq(word, "handle-reboot-key", l))
```

Comment:
18 → 17 or am I miscounting?

Reply:
You are correct, I did not correct it while renaming to reboot.